### DISCUSSION TITLE
WebSocket not sincronizing organizations items

### DISCUSSION THREAD

**Discussion by Basajaun** [dani-garcia/vaultwarden#1376] [Q&A]
### Subject of the issue
WebSocket notifications return error when trying to sync an item of an organization.
Personal items are synced OK

### Your environment

* Bitwarden_rs version: 1.19.0

* Install method: Docker
* Clients used:  Web, deskotp (windows/mac), browser extension (firefox, edge, chrome)
* Reverse proxy and version: pfsense HA Proxy 0.61_1
* Other relevant information:

### Steps to reproduce
Setup websocket them:
Try to create/edit and item in a personal vault (this shoud go fine)
Try to create/edit and item inside a organization (been owner of it or with permission to write on it) (this fails)

### Expected behaviour
Push change every 14 seconds

### Actual behaviour
Error on websocket logs appears

### Relevant logs
[2021-02-09 14:03:27.028][parity_ws::io][INFO] Accepted a new tcp connection from 10.178.38.2:11449.

[2021-02-09 14:03:27.028][parity_ws::handler][ERROR] WS Error <Io(Kind(InvalidData))>: Missing access token

**Comment by BlackDex** [comment]
Part from that WebSocket error which looks like a wrong configuration of HAProxy (See: https://github.com/dani-garcia/bitwarden_rs/wiki/Proxy-examples)
This is a duplicate of: #771

Converting to a discussion because it's a duplicate.
